packages/serverpod_shared/test_integration/assets/load_passwords/runner_missing_passwords.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+import 'package:serverpod_shared/serverpod_shared.dart';
+
+void main() {
```

Comment:
Question: do we really need these two tests in different files? Seems like they are testing the same thing just with different configurations?

Reply:
The reason we need this is because we have to run this code in a different process to be able to inject the environment variables. To run them in a different process we need a main method, hence this file.

Reply:
And I'm too lazy to build some arg parser for this, seems a little overkill for the current need.